bsp/esp-box-3/esp-box-3.c

```diff
@@ -53,8 +53,11 @@ static const ili9341_lcd_init_cmd_t vendor_specific_init[] = {
     {0, (uint8_t []){0}, 0xff, 0},
 };
 
+#if (BSP_CONFIG_NO_GRAPHIC_LIB == 0)
 static lv_display_t *disp;
 static lv_indev_t *disp_indev = NULL;
+#endif // (BSP_CONFIG_NO_GRAPHIC_LIB == 0)
+
 static esp_lcd_touch_handle_t tp;   // LCD touch handle
 static esp_lcd_panel_handle_t panel_handle = NULL;
```

Comment:
Relevant only when LVGL is used. Please move to `#if (BSP_CONFIG_NO_GRAPHIC_LIB == 0)` section